omod/src/main/webapp/pages/visit/visit.gsp

```diff
@@ -40,6 +40,10 @@
     ui.includeJavascript("pihcore", "visit/encounterTransaction.js")
     ui.includeJavascript("pihcore", "services/configService.js")
     ui.includeJavascript("file", "configuration/pih/scripts/visit/encounterTypeConfig.js", /* priority= */ null, /* pathIsRelativeToScripts= */ false)
+
+    if (locale != "en") {
+        ui.includeJavascript("pihcore", "angular-locale/angular-locale_" + locale + ".js")
+    }
 %>
```

Comment:
It seems that the only way to localize the angular-ui datepicker is to load in an appropriate locale file on the page, which sets various constant bindings that the datepicker uses.  I downloaded a french and spanish version of these and added them to the module, and then load these in if necessary.  I left it open-ended in case we need to add additional files for additional locales (presumably it will simply fail gracefully with a 404 if the JS file referenced here doesn't actually exist).